method_example/base_class_clsmethod.py

```diff
@@ -0,0 +1,21 @@
+class BaseClass:
+    
+    _classvariable = None
+    
+    def __init__(self, val):
+        self.instance_varable = val
+        
+    def show_variable(self):
+        print(f"instance variable update direct from init: {self.instance_varable}")
+        
+    def show_class_variable(cls):
+        print(f"class variable update direct from init: {cls._classvariable}")
+        
+    def update_instance_variable(self):
+        self.instance_varable = None
```

Comment:
this is not a meaningful operation, modify this code to make it meaningful. You will not modify the variable with None in most of the cases right?